docs/en/tasks/2d_fashion_landmark.md

```diff
@@ -74,3 +75,64 @@ mmpose
         │   │-- img_00000005.jpg
         │   │-- ...
 ```
+
+## DeepFashion2
+
+<!-- [DATASET] -->
+
+<details>
+<summary align="right"><a href="https://arxiv.org/pdf/1901.07973.pdf">DeepFashion2 (CVPR'2019)</a></summary>
+
+```bibtex
+@article{DeepFashion2,
+  author = {Yuying Ge and Ruimao Zhang and Lingyun Wu and Xiaogang Wang and Xiaoou Tang and Ping Luo},
+  title={A Versatile Benchmark for Detection, Pose Estimation, Segmentation and Re-Identification of Clothing Images},
+  journal={CVPR},
+  year={2019}
+}
+```
+
+</details>
+
+<!-- [DATASET] -->
+
+For [DeepFashion2](https://github.com/switchablenorms/DeepFashion2) dataset, images can be downloaded from [download](https://drive.google.com/drive/folders/125F48fsMBz2EF0Cpqk6aaHet5VH399Ok?usp=sharing).
+Please download the [annotation files](https://drive.google.com/file/d/1RM9l9EaB9ULRXhoCS72PkCXtJ4Cn4i6O/view?usp=share_link). These annotation files are converted by [deepfashion2_to_coco.py](https://github.com/switchablenorms/DeepFashion2/blob/master/evaluation/deepfashion2_to_coco.py).
+Extract them under {MMPose}/data, and make them look like this:
+
+```text
+mmpose
+├── mmpose
+├── docs
+├── tests
+├── tools
+├── configs
+`── data
+    │── deepfashion2
+        │── train
+            │-- deepfashion2_short_sleeved_outwear_train.json
+            │-- deepfashion2_short_sleeved_dress_train.json
+            │-- deepfashion2_skirt_train.json
+            │-- deepfashion2_sling_dress_train.json
+            │-- ...
+            │-- img
```

Comment:
The original data structure is:
```
Training images: train/image Training annotations: train/annos
Validation images: validation/image Validation annotations: validation/annos
Test images: test/image
```
How about keep the name of `image`, instead of renaming it to `img`?

Reply:
> The original data structure is:
> 
> ```
> Training images: train/image Training annotations: train/annos
> Validation images: validation/image Validation annotations: validation/annos
> Test images: test/image
> ```
> 
> How about keep the name of `image`, instead of renaming it to `img`?

I will try

Reply:
Maybe it is a typo?

Reply:
> Maybe it is a typo?

This is my experiment setting problem. I will fix that